### DISCUSSION TITLE
Stack, VStack, HStack

### DISCUSSION THREAD

**Discussion by bobby-alain** [chakra-ui/chakra-ui#3539] [Ideas] [answered]
Right now there's no way to configure Stack Props.

I want all my Stack to look it's children the same, so i don't have to give the spacing prop everytime.

**Comment by chr-ge** [comment]
Can you give some more details or a code snippet? It's unclear what you are asking for.

**Reply by bobby-alain** [reply]
Instead of <Stack space={"8"}>...</Stack> and telling space for the children everytime i use Stack. 
If i could ExtandThem  and something like <myStack>...</myStack>. If that makes sense?

**Comment by chasinhues** [comment, marked as answer]
I don't believe this can be done in the theme currently. I would suggest creating a component that you can reuse that has the common spacing you want applied to it. Something like:

```jsx
import { Stack, StackProps } from "@chakra/react";

export const MyCustomStack = (props: StackProps) => <Stack spacing={8} {...props} />
```

Then you can use it:
```jsx
import { MyCustomStack } from './MyCustomStack';

<MyCustomStack /> // Automatically has spacing={8}
```
